baselines/README.md

```diff
@@ -0,0 +1,47 @@
+# Baseline Testing
+
+This folder contains examples and results for performing testing of baseline controllers for different testcases. 
+
+## Baseline Controller and Test Case Implementation
+
+- The `root/examples/python/controllers/baseline.py` module file is utilized for executing the baseline control for each test case. It simplifies the process by setting `u={}`.
+
+- The test script `run_baselines.py` facilitates the execution of a sample baseline test, with a configuration file `config.json` and the test case name provided as a command-line argument.
+
+## Configuration File Description
+
+- The `config.json` file is used to configure settings for running baseline simulations. It controls options for defining test scenarios, enabling user-defined tests, and saving results. 
+- The `config.json` file includes predefined representative scenarios for all the testcases with different combinations of electricity pricing and time periods. Two-week simulation are conducted with three electricity price schemes and representative time periods for each testcase. For each predefined representative scenario, one week simulation is conducted for the warm-up and two-week simulation (one week before `time_period` and one week after) is then conducted. Please see `root/Testcases/README.md` for different predefined scenarios for each testcase.
+- **`save_kpi_results`**: Save calculated key performance indicator (KPI) results (`true` or `false`).
+- **`save_measurements`**: Save measurement outputs (`true` or `false`).
```

Comment:
Specify where these results are stored upon execution.  If new directory is created, say so and what the path is.

Reply:
Added the following sentence "Upon execution, the measurement outputs are saved in the directory `root/baselines/result`. If the `result` directory doesn't exist, it will be created automatically."